### DISCUSSION TITLE
article/notion-next-gitalk

### DISCUSSION THREAD

**Discussion by giscus** [tangly1024/tangly1024#94] [Announcements]
# article/notion-next-gitalk

NotionNext添加Giscus/Gitalk/Utterance的步骤教程

https://docs.tangly1024.com/article/notion-next-gitalk

**Comment by shijianzhong** [comment]
NEXT_PUBLIC_COMMENT_GITALK_CLIENT_REPO这个变量的值不应该填写tangly1024，应该写仓库名称test-comment
